src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default class Button extends React.Component {
```

Comment:
Since we don't need `state` here nor any lifecycle method, we can convert this safely into a stateless-functional component instead of using `class`.

Reply:
I'm aware stateless components don't need to be classes, but I like it better this way. Personal preference.

Reply:
Ok, **practical experience** wise, this isn't a logical way to do this:

1. If we aren't using `state` and component lifecycle methods, using `class` is just adding unnecessary baggage to this component - `App` is inheriting from `React.Component` which has all those extra unused baggage. 

2. Personal preference is ok but yeah again with **practical experience** should help us understand the more sensible and proper way to do things.